ThermodynamicsSA.pyx

```diff
@@ -184,7 +204,11 @@ cdef class ThermodynamicsSA:
             Py_ssize_t ql_shift = DV.get_varshift(Gr, 'ql')
             Py_ssize_t qi_shift = DV.get_varshift(Gr, 'qi')
             Py_ssize_t qv_shift = DV.get_varshift(Gr, 'qv')
-            Py_ssize_t s_shift = PV.get_varshift(Gr, 's')
+            Py_ssize_t qc_shift
+            Py_ssize_t s_shift
+            Py_ssize_t thli_shift
```

Comment:
This is related to thetali.

Reply:
you mean only 'Py_ssize_t thli_shift' - right ?

Reply:
Yes.